packages/p2p/src/components/advertiser-page/advertiser-page-stats.jsx

```diff
@@ -9,6 +9,8 @@ import './advertiser-page.scss';
 const AdvertiserPageStats = () => {
     const { advertiser_page_store, general_store } = useStores();
 
+    const is_my_advert = advertiser_page_store.advertiser_details_id === general_store.advertiser_id;
+    const info = is_my_advert ? general_store.advertiser_info : advertiser_page_store.counterparty_advertiser_info;
```

Comment:
So I found the cause of this issue: `this.advertiser_details_id` in the advertiser page store is correctly pointing to the clicked advertiser ID, but when we navigate to our own adverts, we make a call to `p2p_advertiser_info` with `subscribe: 1` and the id as our advertiser ID, but this returns an error from the websocket. 
<img width="332" alt="Screenshot 2022-11-17 at 7 13 55 PM" src="https://user-images.githubusercontent.com/103016120/202439726-21ea31e8-a39f-4fe4-aa01-16e9cdeea9b3.png">

This is because we are already subscribed to listen to our own advertiser details within general store, and thus the reason why the websocket is returning `AlreadySubscribed` error (by omitting the `id` field, we are essentially calling advertiser info on our own profile as per the API documentation):
<img width="883" alt="Screenshot 2022-11-17 at 7 55 33 PM" src="https://user-images.githubusercontent.com/103016120/202439898-f2b961ef-4115-48fd-9db6-c416484167ef.png">
So, one of the ways I tried to use a single request was modifying `this.counterparty_advertiser_info` into using `general_store.advertiser_info` if we are looking at our own adverts, but this defeats the purpose of having a counterparty advertiser info as an observable, since it could then contain your own advertiser info or the counterparty advertiser info. Since we are already subscribed to our own `p2p_advertiser_info` endpoint, I feel that we can just reuse our details within `general_store.advertiser_info` **without making unnecessary re-request to again get our personal advertiser info, since its already cached within general store.** If there are any updates to our own advertiser info, **the subscribed endpoint in general store should automatically already re-set our advertiser info** through `general_store.updateAdvertiserInfo`:
<img width="671" alt="Screenshot 2022-11-17 at 8 05 50 PM" src="https://user-images.githubusercontent.com/103016120/202441768-427e87f5-b919-4f9a-a6d7-0ac34447af56.png">

Thus the approach I can find at the moment without breaking other pages using `counterparty_advertiser_info` is to just check within advertiser page, that is:
1. If we are looking at our own advert, use `general_store.advertiser_info`, any subsequent updates to our advertiser info will be updated automatically since we are already subscribed to it in general store (and it would not be unsubscribed even if we leave our advertiser page and check on other user's advertiser info)
2. If we are looking at counterparty adverts, use `advertiser_page_store.counterparty_advertiser_info`
3. Make checks if we are looking at our own advert by `advertiser_page_store.advertiser_details_id === general_store.advertiser_id`

If this approach is not a good one, I could try to create a new computed value to return either `general_store.advertiser_info` or `advertiser_page_store.counterparty_advertiser_info`, but that is essentially the same approach as I am doing within advertiser page